src/gmt_map.c

```diff
@@ -6718,10 +6728,12 @@ void gmt_auto_frame_interval (struct GMT_CTRL *GMT, unsigned int axis, unsigned
 	if (axis == GMT_X) {
 		f = sxy * fabs (GMT->current.proj.rect[XHI] - GMT->current.proj.rect[XLO]);
 		d = fabs (GMT->common.R.wesn[XHI] - GMT->common.R.wesn[XLO]);
+		if (gmtmap_polar_az (GMT)) f *= M_PI_2;	/* Circle perimeter is the right comparison to circular longitudes */
```

Comment:
```suggestion
		if (gmtmap_polar_az (GMT) && d > 350) f *= M_PI_2;	/* Circle perimeter is the right comparison to circular longitudes */
```
In my opinion, `doc/scripts/GMT_stereographic_polar.sh` looked better before the changes. What do you think about limiting these adjustments to only maps that span most longitudes?

Reply:
Yes, that is probably OK, but is 350 the best limit? Did 180 not work?

Reply:
I'll do more tests to determine a reasonable limit. Is it OK for me to push some recommendations to this branch? It would be easy enough to revert them if needed and would work better than GitHub suggestions which are limited to certain lines.

Reply:
Yes, go ahead.